app/components/main-screen.js

```diff
@@ -1,4 +1,8 @@
 import Component from '@ember/component';
+import { reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 
 export default Component.extend({
-});
+  posts: service(),
+  posts: reads('posts.articles')
```

Comment:
articles is incorrect, this is when I tried to create an array of object called articles

Reply:
Also, I confused myself with the naming, not sure what would fit best 😅